app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, new_first_name: str, new_last_name: str):
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            f"VALUES (?, ?)", (new_first_name, new_last_name)
+        )
+        self._connection.commit()
+
+    def all(self):
+        actors_cursor = self._connection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name};"
+        )
+        return [Actor(*row) for row in actors_cursor]
+
+    def update(self, id_: int, new_first_name: str, new_last_name: str):
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET first_name = ?, last_name = ?"
+            f"WHERE id = ?",
+            (id_, new_first_name, new_last_name)
```

Comment:
sorry, missed a mistake here. Incorrect order of parameters, so `update` doesn't work correctly.